webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/policyServerManagement.html

```diff
@@ -174,26 +174,52 @@ <h3 class="page-title">Security</h3>
         </form>
       </div>
     </div>
+  <div class="inner-portlet">
+    <h3 class="page-title">Reporting protocol</h3>
+    <div class="lift:administration.PropertiesManagement.reportProtocolSection" id="reportProtocolForm">
```

Comment:
there need an introduction text about the protocol, that Syslog is the historical one, but in 5.1 HTTPS is available, better, faster, stronger, safer. Or something like that